project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -851,6 +858,286 @@ function c4m_og_replace_owner_build_form_submit($form, &$form_state) {
+    '#title' => t('Start Date'),
+    '#type' => 'date_popup',
+    '#date_format' => $format,
+    '#description' => t('Select the start date'),
+    '#required' => TRUE,
+    '#default_value' => $default_start_date,
+  );
+
+  if (!empty($form_state['input']['end_date']['date'])) {
+    $default_end_date = $form_state['input']['end_date']['date'];
+  }
+  else {
+    $default_end_date = date($format, time());
+  }
+  $form['end_date'] = array(
+    '#title' => t('End Date'),
+    '#type' => 'date_popup',
+    '#date_format' => $format,
+    '#description' => t('Select the end date'),
+    '#required' => TRUE,
+    '#default_value' => $default_end_date,
+  );
+
+  $form['export_content_form_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Content Export'),
+    '#submit' => array('_c4m_og_export_content_form_submit'),
+  );
+
+  $form['export_members_form_submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Members Export'),
+    '#submit' => array('_c4m_og_export_members_form_submit'),
+  );
+
+  $form['#validate'][] = '_c4m_og_form_valid_export_form';
+
+  return $form;
+}
+
+/**
+ * Validation for the group export form.
+ */
+function _c4m_og_form_valid_export_form($form, &$form_state) {
+  $values = $form_state['values'];
+  if ($values['start_date'] > $values['end_date']) {
+    form_set_error('start_date', t('Please select valid dates.'));
+  }
+}
+
+/**
+ * Submit handler.
+ *
+ * Export content from group.
+ */
+function _c4m_og_export_content_form_submit($form, &$form_state) {
+  $values = $form_state['values'];
+  $start_time = strtotime($values['start_date']);
+  $end_time = strtotime("+1day", strtotime($values['end_date']));
+  $group = c4m_og_current_group();
+
+  $query = db_select('og_membership', 'ogm');
+  $query->join('node', 'n', 'n.nid = ogm.etid');
+  $query->join('users', 'u', 'u.uid = n.uid');
+  $query->leftJoin('node_counter', 'nc', 'nc.nid = n.nid');
+  $query->leftJoin('comment', 'c', 'c.nid = n.nid');
+  $query->leftJoin('flag_counts', 'fc', 'n.nid = fc.entity_id AND fc.fid = 2');
+  $query->leftJoin('file_usage', 'fu', 'fu.id = n.nid');
+  $query->leftJoin('file_download_count', 'fd', 'fu.fid = fd.fid');
+  $query->leftJoin('field_data_c4m_source_entity', 'se', 'se.c4m_source_entity_target_id = n.nid');
+  $query->fields('n', array('title', 'type'));
+  $query->fields('u', array('name'));
+  $query->fields('nc', array('totalcount'));
+  $query->groupBy('n.nid');
+  $query->fields('fc', array('count'));
+  $query->addExpression('COUNT(c.cid)', 'comments');
+  $query->addExpression('COUNT(fd.uid)', 'downloads');
+  $query->addExpression('COUNT(se.entity_id)', 'shares');
+  $query->condition('ogm.entity_type', 'node');
+  $query->condition('ogm.gid', $group->nid);
+  $query->condition('n.created', $start_time, '>=');
+  $query->condition('n.created', $end_time, '<=');
+
+  $content = $query
+    ->execute()
+    ->fetchAll();
+
+  if (empty($content)) {
+    return;
```

Comment:
drupal_set_message "There was no content added/updated between <start> and <end>"